src/commands/up.rs

```diff
@@ -143,17 +170,30 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
                 .with_message("Compressing")
                 .with_finish(ProgressFinish::WithMessage("Compressed".into()));
             pg.enable_steady_tick(Duration::from_millis(100));
+            //#endregion
 
             for entry in walked.into_iter().progress_with(pg) {
                 let entry = entry?;
                 let path = entry.path();
+                if path
+                    .components()
+                    .any(|c| ignore_paths.contains(&c.as_os_str()))
```

Comment:
Wondering if this should be `starts_with` instead?

Reply:
It's like that because I wanted to cover cases where there is git repo inside git repo. or in a monorepo where there is node_modules in a sub folder.

Reply:
Sounds good!